example/helm/values-with-pyroscope.yaml

```diff
@@ -0,0 +1,78 @@
+defaults:
+  image:
+    tag: v1.4
+  time_zone: Asia/Tokyo
+  logging:
+    level: debug
+  server_config:
+    metrics:
+      pprof:
+        enabled: true
+    healths:
+      liveness:
+        livenessProbe:
+          initialDelaySeconds: 60
+      readiness:
+        readinessProbe:
+          initialDelaySeconds: 60
+
+discoverer:
+  resources:
+    requests:
+      cpu: 150m
+      memory: 50Mi
+
+manager:
+  index:
+    resources:
+      requests:
+        cpu: 150m
+        memory: 30Mi
+
+gateway:
+  lb:
+    minReplicas: 2
+    maxReplicas: 2
+    resources:
+      requests:
+        cpu: 150m
+        memory: 150Mi
+    ingress:
+      enabled: true
+      annotations:
+        ingress.kubernetes.io/protocol: h2c
+        ingress.kubernetes.io/ssl-passthrough: "true"
+        traefik.protocol: h2c
+        kubernetes.io/ingress.class: traefik
+      host: localhost
+      servicePort: grpc
+    gateway_config:
+      index_replica: 2
+
+agent:
+  # podAnnotations:
+  #   pyroscope.io/scrape: "true"
```

Comment:
why this part is commented?

Reply:
I heard from @hlts2 that these annotations will be set when successful deployment.

Reply:
so should we remove these comment here?

Reply:
we don't have to cuz we will mention it in blog posts or other documents.

Reply:
@kevindiu 
https://github.com/vdaas/vald/pull/1582/files#diff-35aae708480cf8bc95340e5b5dd62d7902cbd1dcdecaf97e654ee413ce7e3418R53